metaflow/plugins/__init__.py

```diff
@@ -12,6 +12,8 @@ def get_plugin_cli(self):
         FLOW_DECORATORS=[],
         STEP_DECORATORS=[],
         ENVIRONMENTS=[],
+        DATA_PROVIDERS={},
```

Comment:
What are `DATA_PROVIDERS`? It seems unused to me but if that's incorrect - could you give a pointer where this is used?

Reply:
Yes, we don't have the notion of `DATA_PROVIDERS` anymore. Let's chuck it.